package.json

```diff
@@ -63,7 +63,7 @@
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.1",
-    "react": "^15.4.2"
+    "react": "^16.12.0"
```

Comment:
What do you think about making this a peer dependency?

Reply:
That sounds like a good idea. Would leaving it like `"^16.12.0"` be fine? Then it will support all of react 16